wca-regulations.md

```diff
@@ -177,10 +177,10 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 5b2) Any repair to a puzzle must not give the competitor an advantage in solving the puzzle. Penalty: disqualification of the attempt (DNF).
     - 5b3) Permitted repairs:
         - 5b3a) If any parts have fallen out or moved out of place, the competitor may place them back.
-        - 5b3b) If, after repairing the puzzle but before the end of the attempt, the competitor finds that the puzzle is unsolvable, they may disassemble and reassemble a maximum of 4 pieces to make the puzzle solvable.
+        - 5b3b) If a competitor repairs the puzzle but finds that the puzzle is unsolvable later during the solve, they may disassemble and reassemble a maximum of 4 pieces to make the puzzle solvable.
```

Comment:
Nit: "the competitor" is more common in Regulations.

Reply:
I agree, "the competitor" would be more consistent.